include/fiction/algorithms/simulation/sidb/energy_distribution.hpp

```diff
@@ -23,14 +23,40 @@
 namespace fiction
 {
 
+/**
+ * This struct stores the energy state of an SiDB layout. The energy state consists of the electrostatic potential
+ * energy and the degeneracy of the state.
+ */
+struct energy_state
```

Comment:
There seems to be an inconsistency now between `energy_state` and `sidb_energy_distribution`. Why is one for SiDBs, but the other one is not? :laughing: The fact that they are in the `simulation/sidb/` folder should be sufficient to no longer require `sidb_` prefixes, or?

Reply:
True! Let's get rid of it